### ISSUE TITLE
add fasttext support

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#79]
https://github.com/facebookresearch/fastText

There is a Python wrapper (https://github.com/salestock/fastText.py), but I'm not sure I like its API, and it segfaults for me sometimes. A time to create another wrapper? :)

**Comment by vinhkhuc** [comment]
Hi @kmike,
fastText now has the official Python wrapper (https://github.com/facebookresearch/fastText/tree/master/python). Is there any plan to support it in eli5? Thanks.
